playbooks/capsule/tasks/main.yaml

```diff
@@ -0,0 +1,65 @@
+---
+- name: 'Set SELinux to Permissive'
+  selinux: policy=targeted state=permissive
+
+- name: 'Get Server Hostname'
+  shell: hostname -f
+  delegate_to: "{{ server }}"
+  register: server_fqdn
+- name: 'Get Server IP Address'
+  shell: ifconfig eth0 2>/dev/null|awk '/inet/ {print $2}'|sed 's/addr://'
```

Comment:
Given ifconfig output changed and isn't designed to be parsed, should IP be used?
```shell
ip -4 address show dev eth0 | awk '/inet/ { print $2 }' | sed 's|/[0-9]\+||'
```